website-next/pages/login.tsx

```diff
@@ -0,0 +1,11 @@
+import { useRouter } from 'next/router'
+import { auth0Url } from '../components/common/env-vars'
+
+export default function Login() {
```

Comment:
I put this here because we had the same routing rule in the `website` but I think this can be deleted as it no longer seems to work

Reply:
Seems like it should go in the bin, but I'm fine for that to happen in a follow up PR